### ISSUE TITLE
Getting exit code 0 regardless of the test results

### ISSUE THREAD

**Issue by kobik** [adamgruber/mochawesome#111]
Hi,

When running tests with v2.0.0, i'm always getting exit code 0, regardless of if the tests have failed or not.
I've verified that in v1.5.4 when the tests failed, the exit code was 2.

**Comment by adamgruber** [comment]
Example code/usage?

**Comment by kobik** [comment]
Here you go
**Test**
```
var should = require('should');

it('test mochawesome', function () {
   true.should.equal(false) 
});
```

**With mochawesome 2.0.0**
```
bash-3.2$ node_modules/mocha/bin/mocha test/mytest.js --reporter mochawesome; echo $?
  1) test mochawesome

  0 passing (14ms)
  1 failing

  1)  test mochawesome:

      AssertionError: expected true to be false
      + expected - actual

      -true
      +false

      at Assertion.fail (node_modules/should/cjs/should.js:231:17)
      at Assertion.value (node_modules/should/cjs/should.js:305:19)
      at Context.<anonymous> (test/mytest.js:4:16)
[mochawesome] Report JSON saved to mochawesome-reports/mochawesome.json

[mochawesome] Report HTML saved to mochawesome-reports/mochawesome.html
0
bash-3.2$
```

**Without mochawesome**
```
bash-3.2$ node_modules/mocha/bin/mocha test/mytest.js ; echo $?
  1) test mochawesome

  0 passing (14ms)
  1 failing

  1)  test mochawesome:

      AssertionError: expected true to be false
      + expected - actual

      -true
      +false

      at Assertion.fail (node_modules/should/cjs/should.js:231:17)
      at Assertion.value (node_modules/should/cjs/should.js:305:19)
      at Context.<anonymous> (test/mytest.js:4:16)
1
bash-3.2$
```

**Comment by bonesoul** [comment]
having the exact same problem.

additionaly generated report file is not rendering with chrome / firefox.

**Comment by bonesoul** [comment]
still same with 2.0.1

**Comment by adamgruber** [comment]
Still looking into it. 2.0.1 was to fix some other issues.

**Comment by adamgruber** [comment]
Should be fixed in 2.0.2
